packages/calcite-components/src/components/stepper-item/stepper-item.e2e.ts

```diff
@@ -14,6 +16,24 @@ describe("calcite-stepper-item", () => {
   });
 
   describe("translation support", () => {
-    t9n(`<calcite-stepper-item heading="Step 1" id="step-1"></calcite-stepper-item>`);
+    t9n(html`<calcite-stepper-item heading="Step 1" id="step-1"></calcite-stepper-item>`);
```

Comment:
Q: do we need `html` util for one liners ?

Reply:
Technically, no. Force of habit. 😅

I'll look into automating this at some point.